front/src/components/Index.jsx

```diff
@@ -25,7 +28,7 @@ const Index = () => {
     <>
       <h1 style={{ color: "blue", fontSize: "18px" }}>Polls</h1>
       {questions.length > 0
-        ? questions.map((item) => listQuestion(item))
+        ? questions.slice(-5).map((item) => listQuestion(item))
```

Comment:
This is a "magic number". What is -5?

Reply:
Just learned what a magic number is and I agree, lets get rid of them. This slice is so that only the 5 most recent questions of the list are mapped. This was a previous requirement from the django polls app before it got brought to react, but is also a good idea to not display every poll because there could be too many for the page to properly display. Although, I’ve now realized that instead of slicing the complete list in react, I should be pulling only the 5 most recent polls from the API instead. As you'll see in the next commit, I’ve created an extra action in the QuestionViewSet called get_queryset that returns the 5 most recent poll questions.